scaffolds/java_actor_model/.gitignore

```diff
@@ -0,0 +1,17 @@
+target/
+
+.settings
+.project
+.classpath
+
+.idea
+*.iml
```

Comment:
I'm ok with this gitignore, but small tip for next time: IDE-specific files to ignore are best put in the [user's global gitignore](https://stackoverflow.com/a/7335487), since it's user-specific and not project specific.

Reply:
Thanks for the tip!